deltaspike/examples/pom.xml

```diff
@@ -51,46 +51,153 @@
     <packaging>pom</packaging>
 
     <name>Apache DeltaSpike Examples</name>
-
+    
     <modules>
-        <!--
         <module>jse-examples</module>
         <module>jsf-examples</module>
-        -->
         <module>jsf-playground</module>
-        <!--
         <module>security-requested-page-after-login-cdi</module>
         <module>security-requested-page-after-login-picketlink</module>
         <module>scheduler-playground</module>
         <module>jpa-examples</module>
         <module>data-examples</module>
-        -->
     </modules>
 
     <properties>
         <deploy.skip>false</deploy.skip>
+        <deltaspike.examples.version>${project.version}</deltaspike.examples.version>
```

Comment:
do we really need this?
it's always ${project.version}, i dont see a reason to wrap this in another property